spec/files/config-with-all-platforms-stage-type.yml

```diff
@@ -0,0 +1,8 @@
+platform_type: Magento
```

Comment:
Is this file being used anywhere?

Reply:
Yes, in a test in `shopify_transporter_spec`